Citavi 5 XML.js

```diff
@@ -0,0 +1,980 @@
+		attachPersons(doc, item, authors, "author");
+		var editors = ZU.xpathText(doc, '//ReferenceEditors/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, editors, "editor");
+		var collaborators = ZU.xpathText(doc, '//ReferenceCollaborators/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, collaborators, "contributor");
+		var organizations = ZU.xpathText(doc, '//ReferenceOrganizations/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		attachPersons(doc, item, organizations, "contributor");
+		
+		var publishers = ZU.xpathText(doc, '//ReferencePublishers/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		if (publishers && publishers.length>0) {
+			item.publisher = attachName(doc, publishers).join('; ');
+		}
+		
+		var keywords = ZU.xpathText(doc, '//ReferenceKeywords/OnetoN[starts-with(text(), "'+item.itemID+'")]');
+		if (keywords && keywords.length>0) {
+			item.tags = attachName(doc, keywords);
+		}
+		if (rememberTags[item.itemID]) {
+			for (var j=0; j<rememberTags[item.itemID].length; j++) {
+				item.tags.push(rememberTags[item.itemID][j]);
+			}
+		}
+		
+		//For all corresponding knowledge items attach a note containing
+		//the information of it.
+		var citations = ZU.xpath(doc, '//KnowledgeItem[ReferenceID="'+item.itemID+'"]');
+		for (var j=0; j<citations.length; j++) {
+			var noteObject = {};
+			noteObject.id = ZU.xpathText(citations[j], '@id');
+			var title = ZU.xpathText(citations[j], 'CoreStatement');
+			var text = ZU.xpathText(citations[j], 'Text');
+			var pages = extractPages(ZU.xpathText(citations[j], 'PageRange'));
+			noteObject.note = '';
+			if (title) {
+				noteObject.note += '<h1>' + title + "</h1>\n";
+			}
+			if (text) { 
+				noteObject.note += "<p>" + ZU.xpathText(citations[j], 'Text') + "</p>\n";
+			}
+			if (pages) {
+				noteObject.note += "<i>" + pages + "</i>";
+			}
+			if (rememberTags[noteObject.id]) {
+				noteObject.tags = rememberTags[noteObject.id];
+			}
+			if (noteObject.note != "") {
+				item.notes.push(noteObject);
+			}
+		}
+		
+		//Locations will be saved as URIs in attachments etc.
+		var locations = ZU.xpath(doc, '//Locations/Location[ReferenceID="'+item.itemID+'"]');
+		for (var j=0; j<locations.length; j++) {
+			var address = ZU.xpathText(locations[j], 'Address');
+			var addressType = ZU.xpathText(locations[j], 'MirrorsReferencePropertyId');
+			if (address) {
+				if (addressType == "Doi" && !item.DOI) {
+					item.DOI = address;
+				} else if (addressType == "PubMedId" && item.extra.indexOf("PMID") == -1) {
+					addExtraLine(item, "PMID", address);
+				} else {
+					item.attachments.push({
+						url: address,
+						title:"Location"
+					});
+				}
+			}
+			var callNumber = ZU.xpathText(locations[j], 'CallNumber');
+			var libraryId = ZU.xpathText(locations[j], 'LibraryID');
+			if (callNumber) {
+				item.callNumber = callNumber;
+				item.libraryCatalog = ZU.xpathText(doc.getElementById(libraryId), "Name");
```

Comment:
By deleting the condition as you suggested, it may (depending on the order the objects are processed) lead to the situation that the call number does not belong to the library saved in Zotero.